tools/README.md

```diff
@@ -98,13 +98,27 @@ bin/prepare_prebuilt_workers \
      -r containers/pre_built_workers
 ```
 
-This builds `cxx` and `go` images contain workers built from the commit/branch
-we wish to test.
+This builds `cxx` and `go` images contain workers built from the specific
+commit/branch we wish to test.
+
+You may also specify the repo where the source code is pulled from by specifying
+the repo. The following command pulls the source code from forked repos
+`${USER}/grpc` and `${USER}/grpc-go` to build cxx and go images
+
+```shell
+bin/prepare_prebuilt_workers \
+     -l cxx:"${USER}"/grpc:master \
```

Comment:
Please put the "" around the whole string, "cxx:${USER}/grpc:master" and "go:${USER}/grpc-go:master".

Reply:
The reason is to prevent splitting?  -- "An argument enclosed in double quotes presents itself as a single word" Thank you.